com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -627,23 +671,36 @@ internal int[] rendererIndexList
             }
         }
 
+        /// <summary>
+        /// Specifies if the Camera Depth Texture should be used for this <c>UniversalRenderPipelineAsset</c>.
```

Comment:
Not exactly sure on the wording but here's some suggestion to be more clear. This is similar to the camera setting that is already documented https://sourcegraph.com/github.com/Unity-Technologies/Graphics/-/blob/com.unity.render-pipelines.universal/Runtime/UniversalAdditionalCameraData.cs?L322-341
```suggestion
        /// When true, the pipeline creates a depth texture that can be read in shaders. The depth texture can be accessed as _CameraDepthTexture. This setting can be overridden per camera.
```